Python/Product/VSCode/AnalysisVsc/LanguageServer.cs

```diff
@@ -354,6 +355,8 @@ public Task<CompletionItem> CompletionItemResolve(JToken token)
         [JsonRpcMethod("textDocument/documentSymbol")]
         public async Task<SymbolInformation[]> DocumentSymbol(JToken token, CancellationToken cancellationToken) {
             await _prioritizer.DefaultPriorityAsync(cancellationToken);
+            // This call is also used by VSC document outline and it needs correct information
+            await WaitForCompleteAnalysisAsync(cancellationToken);
```

Comment:
Both `_server.WorkspaceSymbols` and `_server.DocumentSymbol` call `entry.GetAnalysisAsync` through `GetModuleVariablesAsync` with 50ms timeout. Shouldn't it work by increasing the timeout?

The problem with `_server.WaitForCompleteAnalysisAsync` is that it isn't reliable. It waits sequentially for three queues to become empty but doesn't stop them from adding new elements, so while it wait for `AnalysisQueue` to become empty, `ParseQueue` may get new item that will clean analysis data, which will result in a bug similar to #4560.